### PR TITLE
[DVCSMP-3664] Aeon Minimote: Fix NPE during button event handling if child isn't found

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/aeon-minimote.src/aeon-minimote.groovy [hunk 1]
```diff
@@ -76,10 +76,10 @@ def buttonEvent(button, held) {
 		log.error "Child device $childDni not found"
 	}
 	if (held) {
-		child?.sendEvent(name: "button", value: "held", data: [buttonNumber: 1], descriptionText: "$child.displayName was held", isStateChange: true)
+		if (child) child.sendEvent(name: "button", value: "held", data: [buttonNumber: 1], descriptionText: "$child.displayName was held", isStateChange: true)
```
Comment (tpmanley) [inline devicetypes/smartthings/aeon-minimote.src/aeon-minimote.groovy:79]: Can you explain what this fixes? The code looks equivalent to me

**Comment by tpmanley** [comment]
Please add a jira and the dth name to the PR title

File: devicetypes/smartthings/aeon-minimote.src/aeon-minimote.groovy (hunk 1 above)
Comment (greens) [reply devicetypes/smartthings/aeon-minimote.src/aeon-minimote.groovy:79]: Arguments are evaluated right to left. `child.displayName` gets evaluated, but `child` is null, hence NPE

**Comment by tpmanley** [comment]
Please make the PR title a little more descriptive so we can see at a glance what this is changing. I'd suggest something like `[DVCSMP-3664] Aeon Minimote: Fix NPE during button event handling if child isn't found`.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**